src/main/java/org/jenkinsci/plugins/pubsub/PubsubBus.java

```diff
@@ -92,7 +88,16 @@ public void onChange() {
                 }
             });
         }
-        return pubsubBus;
+    }
+
+    
+    /**
+     * Get the installed {@link PubsubBus} implementation.
+     * @return The installed {@link PubsubBus} implementation, or default
+     * implementation if none are found.
+     */
+    public synchronized static @Nonnull PubsubBus getBus() {
```

Comment:
Any reason you want to keep it `public`? I think its used primarily in this class so private sounds reasonable?

Reply:
Never mind, there was already a public static method, this looks fine.

Reply:
``` synchronized ``` can be removed